src/integrationTest/java/uk/gov/hmcts/reform/ucmc/controllers/BaseIntegrationTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.ucmc.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import uk.gov.hmcts.reform.ucmc.Application;
+
+@SpringBootTest(
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+    classes = Application.class
+)
+@AutoConfigureMockMvc
+public class BaseIntegrationTest {
+
+    protected static final String BEARER_TOKEN = "Bearer let me in";
+
+    @Autowired
+    protected ObjectMapper objectMapper;
+
+    @Autowired
+    protected MockMvc mockMvc;
+
+    protected ResultActions doPost(String auth, String content, String urlTemplate, Object... uriVars)
+        throws Exception {
```

Comment:
could we handle the exception here, so we don't have to add `throws Exception` to every test (it's unnecessary noise imo)?

Reply:
There would be other reasons as well to throw Exception in test methods i.e. `ResultActions.andExpect()` or `ResultActions.andDo(` etc. If we are ok to use `@SneakyThrow` from lombok in our tests only, we can use that to reduce this noise.

Reply:
As discussed replacing this with @SneakyThrows.